orgstats/stats.go

```diff
@@ -62,18 +63,23 @@ func isBlacklisted(blacklist []string, s string) bool {
 	return false
 }
 
-func (s Stats) add(cs *github.ContributorStats) {
+func (s Stats) add(cs *github.ContributorStats, weeks int32) {
 	if cs.Author == nil {
 		return
 	}
 	stat := s[*cs.Author.Login]
 	var adds int
 	var rms int
 	var commits int
+
 	for _, week := range cs.Weeks {
-		adds += *week.Additions
-		rms += *week.Deletions
-		commits += *week.Commits
+		t1 := time.Now()
+		diff := t1.Sub(week.Week.Time)
+		if  weeks == -1 || diff <= (time.Hour * 24 * 7 * time.Duration(rand.Int31n(weeks)) ) {
```

Comment:
I don't get why the random stuff?

maybe we can just keep a count outside the `for`, and increment for each week we pass?